starfish/image/_stack.py

```diff
@@ -38,8 +38,8 @@ def __init__(self, image_partition):
             self.set_slice(indices={Indices.HYB: h, Indices.CH: c, Indices.Z: zlayer}, data=tile.numpy_array)
 
     @classmethod
-    def from_image_stack(cls, image_stack_name_or_url, baseurl):
-        image_partition = Reader.parse_doc(image_stack_name_or_url, baseurl)
+    def from_url(cls, relativeurl, baseurl):
```

Comment:
What's the vision here? Since there's only one implementation, I'd leave this in the `ImageBase` until we have cause to move it to the more specialized class.

Reply:
the next step is to have a generic `from_url` inside ImageBase that builds a stack or a single image.